packages/foreman/foreman/foreman.spec

```diff
@@ -711,6 +713,43 @@ Meta Package to install requirements for Redis caching support
 %files redis
 %{_datadir}/%{name}/bundler.d/redis.rb
 
+%package dynflow-sidekiq
+Summary: Foreman Redis caching support
+Group:  Applications/System
+# start specfile redis Requires
+Requires: %{?scl_prefix}rubygem(sidekiq) >= 5.0
+Requires: %{?scl_prefix}rubygem(sidekiq) < 6
+Requires: %{?scl_prefix}rubygem(gitlab-sidekiq-fetcher)
+# end specfile redis Requires
+Requires: %{name} = %{version}-%{release}
+
+%description dynflow-sidekiq
+Meta Package to install dynflow sidekiq executor support
+
+%files dynflow-sidekiq
+%{_datadir}/%{name}/bundler.d/sidekiq.rb
+%{_unitdir}/%{dynflow_sidekiq_service_name}.service
+%{_datadir}/%{name}/script/dynflow-sidekiq.rb
+
+%install dynflow-sidekiq
+install -Dp -m0644 extras/systemd/%{dynflow_sidekiq_service_name}.service %{buildroot}%{_unitdir}/%{dynflow_sidekiq_service_name}.service
+
+sed -i '/^ExecStart/ s|/usr/local/bin/sidekiq \(.\+\)$|/usr/bin/scl enable tfm "sidekiq \1"|' %{buildroot}%{_unitdir}/%{dynflow_sidekiq_service_name}.service
```

Comment:
Does `scl enable tfm "sidekiq -C something"` work? I always used `scl enable tfm -- sidekiq -C something`

Reply:
We do a similar thing for running Foreman as a service using puma.

Reply:
I took it from there actually.